user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.md

```diff
@@ -116,6 +116,18 @@ This is a Cube-only feature that can only be activated with the argument `ENABLE
 - **Minimum version**: 9.6.7
 - **Estimated Release version**: To be determined
 
+### DataAPI
+
+Enables Data API functionality and scripted connectors.
+
+This displays the Data Sources module in DataMiner Cube and enables the functionality of Data API and Data Sources.
```

Comment:
It seems to me that the second part of this sentence ("enables the functionality of Data API and Data Sources") means the same as the sentence above it. I'd suggest to either remove this, because there's little point in saying the same thing twice in a row, or to change it to a more detailed explanation of what exactly we mean with "the functionality of Data API".

Reply:
The plan is to introduce specific documentation explaining the functionality of Data API. In the future we could link to these pages for the explanation. For now I propose to write "This displays the Data Sources module in DataMiner Cube". Later this could be rewritten with a link to the documentation.